BinanceExchange.API/Websockets/AbstractBinanceWebSocketClient.cs

```diff
@@ -181,7 +188,14 @@ private Guid CreateUserDataBinanceWebSocket(Uri endpoint, UserDataWebSocketMessa
                     }
                 };
             };
+#if NETSTANDARDFEATURE
```

Comment:
NETSTANDARDFEATURE is not a recognized moniker, it should be NETSTANDARD2_0, also note that the TryAdd() method on a dictionary is only available when targetting NETCOREAPP2_0. So in all targetframeworks this solution uses the ContainsKey() approach is required.